scripts/myrial

```diff
@@ -143,6 +146,12 @@ def main(args):
                 raise "Options dot and --plan are incompatible"
 
             print operator_to_dot(processor.get_logical_plan())
+        elif opt.opt_dot:
+            if opt.from_repr:
```

Comment:
The way you use plan is as follows:

`myrial --plan file_containing_raco_plan`

instead of

`myrial file_containing_myrial_program`

Therefore it is an argument to specify input and you should be able to use it with any of the output options (the output options are those in the `group`). Maybe the naming of `--plan` is confusing. It means input plan.